modules/setting/mailer.go

```diff
@@ -71,7 +71,9 @@ func newMailService() {
 		givenHost := sec.Key("HOST").String()
 		addr, port, err := net.SplitHostPort(givenHost)
 		if err != nil {
-			log.Fatal("Invalid mailer.HOST (%s): %v", givenHost, err)
+			if err.Error() != "missing port in address" {
+				log.Fatal("Invalid mailer.HOST (%s): %v", givenHost, err)
+			}
```

Comment:
We don't need the double nesting, you can `&&` concatenate the conditions.
(I would do it if GitHub would be able to fix their suggestion algorithm)

Reply:
Also, does this really fix anything?
To me, this looks, like it simply masks a fatal error and transforms it into a runtime error whenever a mail should be sent, as they don't have a port to send it to anymore.

Reply:
> Also, does this really fix anything? To me, this looks, like it simply masks a fatal error and transforms it into a runtime error whenever a mail should be sent, as they don't have a port to send it to anymore.

When there is no port, later lines will give it a default port.